guvnor-ala/guvnor-ala-build-maven/src/main/java/org/guvnor/ala/build/maven/config/MavenBuildConfig.java

```diff
@@ -34,14 +34,14 @@
     */
     default List<String> getGoals() {
         final List<String> result = new ArrayList<>();
-        result.add( "package" );
+        result.add("package");
         return result;
     }
 
     default Properties getProperties() {
         final Properties result = new Properties();
-        result.setProperty("failIfNoTests", "false" );
+        result.setProperty("failIfNoTests",
+                           "false");
```

Comment:
@ge0ffrey I already did that couple of weeks ago https://github.com/kiegroup/optaplanner-wb/pull/179